src/routes/delegates/models.rs

```diff
@@ -12,25 +13,25 @@ pub struct Delegate {
 #[derive(Serialize, Deserialize)]
 #[serde(rename_all = "camelCase")]
 pub struct DelegateCreate {
-    safe: Option<String>,
-    delegate: String,
-    delegator: String,
-    signature: String,
-    label: String,
+    pub safe: Option<String>,
+    pub delegate: String,
+    pub delegator: String,
+    pub signature: String,
+    pub label: String,
```

Comment:
We should not change the field visibility just for the tests. I will double check how this module is structured because the test should have access to the private fields.